library/components/TUrlCopyButton.vue

```diff
@@ -1,26 +1,25 @@
 <template>
-    <div class="relative" @mouseover="showTooltip = true" @mouseleave="showTooltip = false">
+    <div class="relative">
         
-        <t-tooltip v-if="showTooltip" position="left" width="max-content">
+        <t-tooltip position="bottom" width="130px">
+            <button
+                class="h-[30px] w-[30px] rounded-[4px] flex items-center justify-center border border-[#C3C2CB] p-1 transition-colors"
+                @click="copy"
+                :class="{ 'bg-[#B4E4D9]': copied }"
+                slot="trigger"
+            >
+                <check-icon :size="18" v-if="copied" />
+                <link-icon :size="18" v-else />
+            </button>
```

Comment:
NON-BLOCKING SUGGESTION:
### Accessibility improvement

This button is an icon only, but has text that we could integrate. We could improve its accessibility by moving the text into the button and hiding it

```suggestion
            <button
                class="h-[30px] w-[30px] rounded-[4px] flex items-center justify-center border border-[#C3C2CB] p-1 transition-colors"
                @click="copy"
                :class="{ 'bg-[#B4E4D9]': copied }"
                slot="trigger"
            >
                <check-icon :size="18" v-if="copied" />
                <link-icon :size="18" v-else />
                <span class="visually-hidden">{{ copied ? "URL Copied" : "Copy URL" }}</span>
            </button>
```


```
/* 
 * Utility class to hide content visually while keeping it screen reader-accessible.
 * Source: https://www.scottohara.me/blog/2017/04/14/inclusively-hidden.html 
 */

.visually-hidden:not(:focus):not(:active) {
  clip: rect(0 0 0 0); 
  clip-path: inset(100%); 
  height: 1px; 
  overflow: hidden; 
  position: absolute; 
  white-space: nowrap; 
  width: 1px; 
}
```

This technique comes from this article: https://www.sarasoueidan.com/blog/accessible-icon-buttons/#technique-%231%3A-accessible-visually-hidden-text

There are several other techniques, but this seems like the most straight forward one for the moment. We could add the `visually-hidden` css utility to our global stylesheets for future use.